src/entity/UserFactory.java

```diff
@@ -1,4 +1,5 @@
 package entity;
 
 public interface UserFactory {
```

Comment:
Follows interface segregation design principle by not defining irrelevant methods.